README.md

```diff
@@ -8,46 +8,49 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/slint-ui/slint)](https://api.reuse.software/info/github.com/slint-ui/slint)
 [![Discussions](https://img.shields.io/github/discussions/slint-ui/slint)](https://github.com/slint-ui/slint/discussions)
 
-Slint is a toolkit to efficiently develop fluid graphical user interfaces for any display: embedded devices and desktop applications. We support multiple programming languages, such as
-Rust, C++, and JavaScript.
+Slint is a toolkit to efficiently design, develop and deploy responsive apps on any screen. Frontends built with Slint can be easily integrated with backends written in different programming languages such as C/C++, Rust, and JavaScript. Slint is developed by SixtyFPS GmbH together with the community. We invite you to use Slint and be part of its community.
```

Comment:
Interesting introduction of frontend and backend here.

Reply:
Open to changes to wording :)

Reply:
Re-reading this, I think the previously sentence was simpler and easier to read. Introducing terms such as "backend" and "frontend" seems tempting from a technical perspective, but it also creates ambiguity for people coming from the web for example. Does this mean that the backend runs on a server and Slint UI runs elsewhere? (traditionally there's a host boundary between those).

Since this is the very first sentence of the README, simplicity and clarify wins over the "feature" that we support a frontend and backend separation. If anything, I think that could be a bullet point under "features" somewhere.

I suggest to go back to the old wording.

Reply:
Maybe Design vs. Logic? 

```suggestion
Slint is a toolkit to efficiently design, develop and deploy responsive apps on any screen. User interface design built with Slint can be easily integrated with logic written in different programming languages such as C++, Rust, and JavaScript. Slint is developed by SixtyFPS GmbH together with the community. We invite you to use Slint and be part of its community.
```

I'm also unsure if we should mention SixtyFPS GmbH so early in the introduction.